test/views/declaration/CommodityMeasureViewSpec.scala

```diff
@@ -115,8 +115,8 @@ class CommodityMeasureViewSpec extends AppViewSpec with CommodityMeasureMessages
 
       val view = createView(CommodityMeasure.form().fillAndValidate(CommodityMeasure(Some("99.99"), "", "10.00")))
 
-      checkErrorsSummary(view)
-      checkErrorLink(view, 1, netMassEmpty, "#netMass")
+      view.getElementById("error-summary-heading").text() mustNot be(empty)
+      view.getElementById("netMass-error").attr("href") mustBe "#netMass"
```

Comment:
same here - `haveHref` but i think it would be nice to add here also something `haveHrefToAnchor` with additional check to if document contains specified id